dspace/modules/additions/src/main/java/org/dspace/authority/AuthorityValueFinder.java

```diff
@@ -99,6 +99,11 @@ public AuthorityValue findByOrcidID(Context context, String orcid_id) {
         return find(context, queryString);
     }
 
+    public List<AuthorityValue> findAllFunders(Context context) {
```

Comment:
Please refactor to "findByAuthorityType(Context context, String type)" which is more reusable. Then call it like valueFinder.findByAuthorityType(context, FunderAuthority.getAuthorityType()).